src/Apps/Search/SearchApp.tsx

```diff
@@ -60,9 +62,7 @@ export class SearchApp extends React.Component<Props> {
     return (
       <>
         <Row>
-          <Col>
-            <RecentlyViewed />
-          </Col>
+          <Col>{/*<RecentlyViewed />*/}</Col>
```

Comment:
TODO: This was just temporarily disabled in order to focus on getting the first search page to render.